client/gutenberg/editor/layout/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { parse } from '@wordpress/blocks';
+import { compose } from '@wordpress/compose';
+import { withDispatch, withSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import BrowserURL from './browser-url';
+import EditorRevisionsDialog from 'post-editor/editor-revisions/dialog';
+
+export class GutenlypsoLayout extends Component {
```

Comment:
I think it may help to add a hint either in a readme, or comment, on where to best put customizations for post-edit.